backend/src/main/java/codezap/template/service/TemplateService.java

```diff
@@ -136,27 +135,39 @@ public FindAllTemplatesResponse findAllBy(
         pageable = PageRequest.of(pageable.getPageNumber() - 1, pageable.getPageSize(), pageable.getSort());
 
         if (categoryId != null && tagIds != null) {
-            List<Long> templateIds = templateTagRepository.findAllTemplateIdInTagIds(tagIds, tagIds.size());
+            List<Long> templateIds = findTemplateIdContainsTagIds(tagIds);
+            categoryRepository.fetchById(categoryId);
```

Comment:
짱수의 의견과 동일합니다.
처음 해당 메서드를 봤을 때 오타인가? 라고 생각하게 되더라구요.
존재 유무를 확인하기 위한 메서드라면 아래와 같은 코드는 어떤가요?
```suggestion
            if (!categoryRepository.existsById(categoryId) {
                throw new CodeZapException(HttpStatus.BAD_REQUEST, "존재하지 않는 카테고리를 조회하였습니다.");
            }
```

Reply:
굿쟙입니다 ~!!